README.rst

```diff
@@ -94,6 +94,8 @@ file of this repository, and can be defined in your project's ``conf.py`` via
         'display_version': True,
         'prev_next_buttons_location': bottom,
         'style_external_links': False,
+        'versioning' : False,
+        'READTHEDOCS' : False,
```

Comment:
This isn't the same thing as RTD is doing. This will set a theme variable, and not the actual HTML context.

Reply:
Yet, `versioning` will require almost the same context data as `READTHEDOCS`: `current_version` (string), `versions` (dictionary), and `downloads` (dictionary). It seems not coherent to handle `READTHEDOCS` as a context field, along with auxiliary fields; but make `versioning` be a theme option, provided that the same context fields are required.